pkg/kube/client.go

```diff
@@ -163,7 +162,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		return "", err
 	}
 
-	var objPods = make(map[string][]core.Pod)
+	var objPods = make(map[string][]v1.Pod)
```

Comment:
This breaks the printing of pod status. The printer only outputs resource-specific information when the internal type is used. With this change, only the age will be printed for pods.

Ideally the internal types shouldn't be used in the helm codebase at all, but for now it is the only way to get the proper output from the printer. There is an ongoing effort to fix this in the kubernetes codebase, but it probably won't be coming until 1.14.

Reply:
objects are converted to internal api before printing